contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -186,6 +192,11 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
   /// @param donId The ID of the removed DON
   event DONRemoved(uint32 donId);
 
+  /// @notice This event is emitted when a new DON is created
+  /// @param donId The ID of the updated DON
+  /// @param isPublic True if the updated DON is public
+  event DONUpdated(uint256 donId, bool isPublic);
```

Comment:
We currently emit this and `DONAdded` but I'm starting to think that it's redundant as we already emit `ConfigSet`.  The listener can derive whether or not the DON was added/updated using the `configCount`.